js/admission.js

```diff
@@ -0,0 +1,134 @@
+
+
+$(document).ready(function(){
+    const noResultBlock= document.querySelector(".noResultBlock");
+    loadData();
+    function loadData(){
+        courseData = getData();
+        populateUI(courseData);
+    }
+    function getData(){
+        var courses = [
```

Comment:
no need to define variable here you can just return array

Reply:
also, as long as there is no logic to get data, you do not really need this function. just define const for this data and call populateUI with it initially